src/metabase/util/encryption.clj

```diff
@@ -75,7 +84,9 @@
        (catch Throwable e
          (if (u/base64-string? s)
            ;; if we can't decrypt `s`, but it *is* encrypted, log an error message and return `nil`
-           (log/error "Cannot decrypt encrypted details. Have you changed or forgot to set MB_ENCRYPTION_SECRET_KEY?" (.getMessage e))
+           (log/error
+            (trs "Cannot decrypt encrypted credentials. Have you changed or forgot to set MB_ENCRYPTION_SECRET_KEY?")
+            (.getMessage e))
```

Comment:
Looks like you just reformatted this, but it is discarding the stacktrace.

Reply:
good point I will put that in